pyscriptjs/src/remote_interpreter.ts

```diff
@@ -30,6 +33,13 @@ type PATHInterface = {
     dirname(path: string): string;
 } & ProxyMarked;
 
+type PyScriptPyInterface = ProxyMarked & {
+    _set_version_info(ver: string): void;
+    uses_top_level_await(code: string): boolean;
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    _run_pyscript(code: string, id?: string): { result: any };
```

Comment:
random comment, but I still think we should tell `eslint` to allow `any`.

Reply:
Based on this:
```
$ rg -g'*.ts' '@typescript-eslint/[a-z-]*' --only-matching --no-line-number --no-filename | sort | uniq -c | sort -h --rev
     10 @typescript-eslint/no-explicit-any
      9 @typescript-eslint/no-unsafe-assignment
      3 @typescript-eslint/no-unused-vars
      1 @typescript-eslint/restrict-template-expressions
      1 @typescript-eslint/no-unsafe-member-access
      1 @typescript-eslint/no-unsafe-call
```
I vote we turn off no-explicit-any and no-unsafe-assignment and leave the rest.

Reply:
Opened a separate PR for this: #1335